docker/example.env

```
@@ -0,0 +1,77 @@
+## Common Docker configuration
+
+# Service broker
+# SERVICE_BROKER=redis://redis_common:6379/0
+
+# Lisk Service log config
+# SERVICE_LOG_CONSOLE=false
+# SERVICE_LOG_FILE=false
+# SERVICE_LOG_GELF=false
+# SERVICE_LOG_LEVEL=debug
+# SERVICE_LOG_STDOUT=true
+
+# Lisk Service Core
+# SERVICE_CORE_REDIS=redis://redis_core_persistent:6379/0
+# SERVICE_CORE_REDIS_VOLATILE=redis://redis_core_volatile:6379/0
+# SERVICE_CORE_MYSQL=mysql://lisk:password@mysql_core:3306/lisk
+
+# Lisk static assets, ie. known account lists
+LISK_STATIC=https://static-data.lisk.com
+
+# Lisk Service geolocation backend
+GEOIP_JSON=
+
+# Create search index (0 = full index)
+INDEX_N_BLOCKS=0
+
+# Use default genesis block height
+# GENESIS_HEIGHT=0
+# FEE_EST_DEFAULT_START_BLOCK_HEIGHT=0
+
+# Fee estimator
+ENABLE_FEE_ESTIMATOR_QUICK=true
+ENABLE_FEE_ESTIMATOR_FULL=false
+
+# Transaction statistics
+ENABLE_TRANSACTION_STATS=true
+
+# Missing blocks job
+ENABLE_JOB_MISSING_BLOCKS=true
+INDEX_MISSING_BLOCKS_RANGE=0
+
+# Log debug messages
+#DEBUG=*dynamicFees*
+
+# # Lisk ecosystem configuration
+LISK_CORE_WS=ws://host.docker.internal:4004
```

Please consider setting it to the default Lisk Core WS port
```suggestion
LISK_CORE_WS=ws://host.docker.internal:8080
```